submissions/Nemooochka/movie-seats/style.css

```diff
@@ -0,0 +1,194 @@
+@import 'https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css';
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap");
+* {
+  box-sizing: border-box;
+}
+
+body {
+  color: #333;
+  font-size: 16px;
+  font-family: "Montserrat", sans-serif;
+}
+
+header {
+  margin: 30px 0 40px;
+  text-align: center;
+}
+
+.container {
+  max-width: 1140px;
+  margin: 0 auto;
+  padding: 0 15px;
+}
+
+.btn {
```

Comment:
Add styles on `:hover`, `:active`, and `:focus` state.
Also, add `cursor: pointer;`

Reply:
Not fixed yet.